client/webserver/locales/pl-pl.go

```diff
@@ -40,7 +40,7 @@ var PlPL = map[string]string{
 	"export_app_pw_msg":              "Podaj hasło aplikacji, aby potwierdzić eksport konta dla",
 	"Disable Account":                "Zablokuj konto",
 	"disable_app_pw_msg":             "Podaj hasło aplikacji, aby zablokowac konto",
-	"disable_irreversible":           `<span class="red">Uwaga:</span> Ta czynność jest nieodwracalna - zablokowanego konta nie można później odblokować.`,
```

Comment:
Are you using google translate for these?  I will ping the people who did the original translations assuming you don't speak pl, pt, and zh.  :)

Reply:
yeah, I should have mentioned that its from Google translate :)

just didn't think you have dedicated people to keep it updated, by all means, let them know and I'll incorporate their feedback here

Reply:
@artikozel how does this PL translation look?

Reply:
Apologies for taking so long! I'll need to make a habit out of checking my github e-mail, and soon!

Yeah, the highlighted string looks and feels okay, which is weird knowing where it came from, but if it's not broken I'm definitely not going to try and fix it.

Reply:
Sounds good @artikozel.  Thanks for verifying!